tests/assembly/nvptx-kernel-abi/nvptx-kernel-args-abi-v7.rs

```diff
@@ -242,22 +242,6 @@ pub unsafe extern "ptx-kernel" fn f_float_array_arg(_a: [f32; 5]) {}
 //pub unsafe extern "ptx-kernel" fn f_u128_array_arg(_a: [u128; 5]) {}
 
 // CHECK: .visible .entry f_u32_slice_arg(
-// CHECK: .param .u64 f_u32_slice_arg_param_0
-// CHECK: .param .u64 f_u32_slice_arg_param_1
+// CHECK: .param .align 8 .b8 f_u32_slice_arg_param_0[16]
```

Comment:
We don't guarantee any ABI for wide pointers anyway and if this is breaks the ABI all users can be updated to pass `#[repr(C)] struct Slice<'a, T> { ptr: *const (), len: usize, _marker: PhantomData<&'a [T]> }` on the host side anyway (and preferably also the gpu side to avoid depending on the unstable ABI).